File: src/ui/app/root/application/models/TimeTable/dayOff.ts
export interface IDayOff {
    dayOffId: string;
    timeTableId: string;
    fromDate: Date;
    toDate: Date;
    statusId: number;
}
export interface IDayOffVM {
    dayOffId: string;
    timeTableId: string;
    fromDate: Date;
    toDate: Date;
    statusId: number;
    fullName:string;
}

File: src/data/Model/Cms360/TimeTable/DayOff.cs

using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Cms360.Data.Model

{
    [Table("DayOff", Schema = "TimeTable")]

    public class DayOff
    {
        [Key]
        public Guid DayOffId { get; set; }


        public Guid TimeTableId { get; set; }

        public DateTime FromDate { get; set; }


        public DateTime ToDate { get; set; }

        public int StatusId { get; set; }

    }


    public class DayOffVM
    {
        [Key]
        public Guid DayOffId { get; set; }


        public Guid TimeTableId { get; set; }

        public DateTime FromDate { get; set; }


        public DateTime ToDate { get; set; }

        public int StatusId { get; set; }
        public string FullName { get; set; }


    }
}